### PR TITLE
Version numbers

### LINKED ISSUE #592: Tidy version numbers
We currently display an app version ( sourced from `./client/package.json` ) and an API version ( sourced from static text in `./server/graphql/general/src/lib.rs` )

Given that the intention is to only host the app from the server, I'm thinking that we should remove some duplication and 

- remove version from the top level `package.json`
- update version in the server manifest `./server/server/Cargo.toml`
- update version in `./client/package.json`
- only display the server version on the settings page

Am still wondering if it is helpful to have a client version displayed - I'm thinking that this will only create confusion, and so should remove it. Something has gone terribly wrong if we have divergent server & client versions

### PR DESCRIPTION
**Pull request by mark-prins**
Closes #592 

Not perfect, but an improvement.
Updated the server and client version numbers, and sourcing the displayed version from the graphql/general package. Ideally we wouldn't need to duplicate the version in `graphql/general/Cargo.toml` and `server/server/Cargo.toml` but I thought it was nice to have the version shown come from the API and to have the server marked with the correct version.

### CODE REVIEW

File: server/graphql/lib.rs [hunk 1]
```diff
@@ -291,34 +291,36 @@ async fn graphql<
...
-        assert_graphql_query!(&settings, &query, &None, expected, None);
-    }
-}
+//         let query = r#"
+//         query {
+//             apiVersion
+//         }
+//         "#;
+
+//         assert_graphql_query!(&settings, &query, &None, expected, None);
```
Comment (andreievg) [inline server/graphql/lib.rs:324]: I think we can just remove this test entirely

File: client/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "openmsupply-client",
-  "version": "1.0.5",
+  "version": "1.0.0-alpha.6",
```
Comment (andreievg) [inline client/package.json:3]: I wonder if we can just comment that version is in server/cargo.toml (or is it required in package.json ?)

**Review by andreievg — APPROVED** [review]
Looks, good, happy for merge.

`note`:
It would be great to have just one place for version. We can remove apiVersion graphql for now and keep tomls on static version and use omsupply-client package.json to display version in f/e

**Comment by Chris-Petty** [comment]
Changes look straight forward! Just thinking, does this address the android app versioning? I've tried to look in the repo for how android builds are done, but spotting I've found is here:

https://github.com/openmsupply/open-msupply/blob/9877e034c450760bb9ab25acec97f31a02ec8242/client/packages/android/app/build.gradle#L9-L10

Inspiration from ol' mobile, we've got some gradle code that slurped the info from the `package.json` to make sure the app versions were bumped along automatically:

https://github.com/openmsupply/mobile/blob/80824641c7e16273c75d5750d70f2e13d5e02de9/android/app/build.gradle#L118-L133

Caution to the wind of the [versionCode having a hard limit of 2100000000](https://developer.android.com/studio/publish/versioning)

**Comment by mark-prins** [comment]
thanks @Chris-Petty - haven't tidied up the android versioning yet. I thought to look at that when working on the printing issue.

**Pull request merged**